### DISCUSSION TITLE
Style for whole MediaWikiTable

### DISCUSSION THREAD

**Discussion by bukowski12** [thombashi/pytablewriter#43] [Q&A] [answered]
Hi
is possible add style for mediawiki table?
i need     {| class="wikitable" style="display: inline-table;"

**Comment by thombashi** [comment, marked as answer]
@bukowski12 
Hi,
You can set style via `table_style` attribute of `MediaWikiTableWriter` class since `pytablewriter 0.62.0`:

```py
import pytablewriter as ptw

writer = ptw.MediaWikiTableWriter(
    headers=["hoge", "foo"],
    value_matrix=[
        [1, "abc"],
        [2, "xyz"],
    ],
    table_style="display: inline-table;",
)
writer.write_table()
```

```
{| class="wikitable" style="display: inline-table;"
! hoge
! foo
|-
| style="text-align:right"| 1
| abc
|-
| style="text-align:right"| 2
| xyz
|}
```

note: This is an experimental feature for now. The interface may change in the future.
